tasks/ldap.yml

```diff
@@ -2,26 +2,26 @@
 # vim: foldmarker=[[[,]]]:foldmethod=marker
 
 - name: Check default ldap config exists
-  command: php --file "{{ owncloud__home }}/occ" ldap:show-config ""
+  command: '{{ owncloud__occ_bin_file_path | quote }} ldap:show-config'
   changed_when: False
   when: (not ansible_check_mode)
   register: owncloud__register_ldap_default_config_exits
   become_user: '{{ owncloud__user }}'
```

Comment:
Well, we need to find a better solution for `owncloud__nginx__php_status_location` but then, sure :wink: